src/api/app/controllers/webui/cloud/static_pages_controller.rb

```diff
@@ -0,0 +1,17 @@
+module Webui
+  module Cloud
+    class StaticPagesController < WebuiController
```

Comment:
To make it more REST I would call this controller ``ConfigurationsController`` and the action ``index``